File: Leetcode/217.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        map<int, int> m;

        for (auto& n : nums) {
            if (m[n] == 0) {
                m[n]++;
                continue;
            }
            return true;
        }

        return false;
    }
};

File: Leetcode/217.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashMap<Integer, Integer> map = new HashMap<Integer, Integer>();

        for (int num : nums) {
            if (map.containsKey(num)) return true;
            map.put(num, 1);
        }

        return false;
    }
}